test/runtests.jl

```diff
@@ -338,11 +333,6 @@ end
             fits_write_pix(f, a)
             close(f)
 
-            f2 = fits_clobber_file(tempname() * ".fits")
-            fits_create_img(f2, eltype(a), [size(a)...])
-            fits_write_pix(f2, a)
-            close(f2)
```

Comment:
Why was this removed?

Reply:
This was a duplicate occurrence of the same code. The lines https://github.com/JuliaAstro/CFITSIO.jl/blob/e4fe70f7c5b408b0bfd784790cb3ee2c354c6d16/test/runtests.jl#L341-L344 and https://github.com/JuliaAstro/CFITSIO.jl/blob/e4fe70f7c5b408b0bfd784790cb3ee2c354c6d16/test/runtests.jl#L405-L409 essentially did the same thing, so I've cleaned it up.